package-lock.json

```diff
@@ -10534,8 +10534,8 @@
       "integrity": "sha512-6qE4B9deFBIa9YSpOc9O0Sgc43zTeVYbgDT5veRKSlB2+ZuHNoVVxA1L/ckMUayV9Ay9y7Z/SZCLcGteW9i7bg=="
     },
     "mattermost-redux": {
-      "version": "github:mattermost/mattermost-redux#477150436ac1871d0d63954c4f4f4d16ff692483",
-      "from": "github:mattermost/mattermost-redux#477150436ac1871d0d63954c4f4f4d16ff692483",
+      "version": "github:mattermost/mattermost-redux#aa10e21b4938adf580c6be4480819008e1a33eac",
```

Comment:
I think this should be done after the mattermost-redux get merged into master

Reply:
Agree, just added this so the tests would pass